### DISCUSSION TITLE
Result return by environment variable

### DISCUSSION THREAD

**Discussion by phmonte** [stryker-mutator/stryker-net#2344] [Ideas]
Does it make sense to implement a return of the result through an environment variable and/or file?

Scenario:
I run the tests inside a docker container, to use stryker in order to stop the pipeline it would be necessary to capture the output of the container.

If it makes sense, I can implement and submit for validation.

**Comment by rouke-broersma** [comment]
That seems like a misuse of environment variables. I would think a better solution would be to return a nonzero exit code from the container if stryker returns a nonzero exit code (threshold break).
